cavern/src/main/java/org/opencadc/cavern/CavernConfig.java

```diff
@@ -169,6 +184,19 @@ public CavernConfig() {
             allocationParents.add(ap);
         }
 
+        adminAPIKeys.addAll(mvp.getProperty(CavernConfig.ADMIN_API_KEY));
+
+        final String configuredDefaultQuotaGB = mvp.getFirstPropertyValue(CavernConfig.DEFAULT_QUOTA_GB);
+        if (configuredDefaultQuotaGB == null) {
+            throw new InvalidConfigException(CavernConfig.DEFAULT_QUOTA_GB + " must be specified");
```

Comment:
default quota must be optional so the only invalid config is a failure to parse as a double (which you have below).